docker-compose.yml

```diff
@@ -20,311 +20,342 @@ x-http-proxy: &http-proxy ${HTTP_PROXY}
 x-https-proxy: &https-proxy ${HTTPS_PROXY}
 x-no-proxy: &no-proxy localhost,0.0.0.0,127.0.0.1,uclvlddpragae07,hasher-api,orthanc-raw
 x-proxy-common: &proxy-common
-  HTTP_PROXY: *http-proxy
-  http_proxy: *http-proxy
-  HTTPS_PROXY: *https-proxy
-  https_proxy: *https-proxy
-  NO_PROXY: *no-proxy
-  no_proxy: *no-proxy
+    HTTP_PROXY: *http-proxy
```

Comment:
these are a lot of tabs that I'm guessing weren't intentional. Don't have strong opinions on spacing but we probably don't want to be mixing tabs and spaces

Reply:
Ah crap, that's my autoformatter (though I do like 4-space indents)
But I don't see any tabs, just 4-space indents?

Reply:
I think you've just switched it from 2 to 4 spaces. Mine does it too, and I'd argue it's the standard/cleaner, but we should keep it the same across all our yml files